src/internal/converters/ics/ics.go

```diff
@@ -192,108 +199,148 @@ func getRecurrencePattern(
 }
 
 func FromJSON(ctx context.Context, body []byte) (string, error) {
-	data, err := api.BytesToEventable(body)
+	event, err := api.BytesToEventable(body)
 	if err != nil {
 		return "", clues.Wrap(err, "converting to eventable")
 	}
 
 	cal := ics.NewCalendar()
 	cal.SetProductId("-//Alcion//Corso") // Does this have to be customizable?
 
-	id := data.GetId() // XXX: iCalUId?
-	event := cal.AddEvent(ptr.Val(id))
+	id := ptr.Val(event.GetId())
+	iCalEvent := cal.AddEvent(id)
 
-	created := data.GetCreatedDateTime()
+	err = updateEventProperties(ctx, event, iCalEvent)
+	if err != nil {
+		return "", clues.Wrap(err, "updating event properties")
+	}
+
+	exceptionOcurrances := event.GetAdditionalData()["exceptionOccurrences"]
+	if exceptionOcurrances != nil {
+		for _, occ := range exceptionOcurrances.([]any) {
+			instance, ok := occ.(map[string]any)
+			if !ok {
+				return "", clues.New("converting exception instance to map[string]any").
+					With("type", fmt.Sprintf("%T", instance))
+			}
+
+			exBody, err := json.Marshal(instance)
+			if err != nil {
+				return "", clues.Wrap(err, "marshalling exception instance")
```

Comment:
WrapWC is needed since json.Marshal doesn't handle Ctx addition.  This goes for some errors below as well.
```suggestion
				return "", clues.WrapWC(ctx, err, "marshalling exception instance")
```

Reply:
I've updated this and all other instances where we have context available to use `WC` versions of clues functions.